### PR TITLE
Added salad.css

### PR DESCRIPTION
**Pull request by OugiFormula**

### CODE REVIEW

**Comment by lxjv** [comment]
sald

**Comment by OugiFormula** [comment]
sald

File: profiles/themes/Salad.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+
+Theme: Salad
+Author: YukioKoito
+Version: 1.0
+Description: salad theme after the salad.com color scheme
```
Comment (Vukkyy) [inline profiles/themes/Salad.css:6]: questionable brand placement

**Review by Vukkyy — CHANGES_REQUESTED** [review]

File: profiles/themes/Salad.css (hunk 1 above)
Comment (OugiFormula) [reply profiles/themes/Salad.css:6]: this is their color scheme and I used it, reasonable to mention them there.

Comment (lxjv) [reply profiles/themes/Salad.css:6]: L

**Pull request closed without merging**